package/MDAnalysis/coordinates/PDB.py

```diff
@@ -600,23 +608,24 @@ def _read_frame(self, frame):
             for line in f:
                 if line[:6] == 'ENDMDL':
                     break
-                # NOTE - CRYST1 line won't be found if it comes before the MODEL
-                # line, which is sometimes the case, e.g. output from gromacs
-                # trjconv
+                # NOTE - CRYST1 line won't be found if it comes before the
+                # MODEL line, which is sometimes the case, e.g. output from
+                # gromacs trjconv
                 elif line[:6] == 'CRYST1':
-                    A, B, C = map(float, [line[6:15], line[15:24], line[24:33]])
-                    alpha, beta, gamma = map(float, [line[33:40], line[40:47],
+                    A, B, C = np.float32([line[6:15], line[15:24],
+                                          line[24:33]])
+                    alpha, beta, gamma = np.float32([line[33:40], line[40:47],
                                                      line[47:54]])
                     self.ts._unitcell[:] = A, B, C, alpha, beta, gamma
                     continue
                 elif line[:6] in ('ATOM  ', 'HETATM'):
                     # we only care about coordinates
-                    self.ts._pos[pos] = list(map(float, [line[30:38],
-                                                         line[38:46],
-                                                         line[46:54]]))
+                    self.ts._pos[pos] = np.float32([line[30:38],
```

Comment:
So I'm not saying that `list(map(float` is pretty, but, doesn't this create an intermediate np.array for every coordinate?